test/IDE/Inputs/custom-modules/ImportAsMemberError.h

```diff
@@ -19,15 +19,13 @@ typedef NSObject<ImportedProtocolBase> *ImportedProtocolBase_t;
 typedef NSObject<ErrorProto> *ErrorProto_t;
 
 // Instance and static member onto protocol
-void mutateSomeStaticState()
+void mutateSomeStaticState(void)
     __attribute__((swift_name("ErrorProto.mutateSomeStaticState()"))); // ok
 void mutateSomeInstanceState(ErrorProto_t self) __attribute__((
     swift_name("ErrorProto.mutateSomeInstanceState(self:)"))); // error
 
 // Non-prototype declaration
 extern void IAMErrorStructHasPrototype(void)
     __attribute__((swift_name("ErrorStruct.hasPrototype()"))); // ok
-extern void IAMErrorStructNonPrototype()
-    __attribute__((swift_name("ErrorStruct.nonPrototype()"))); // error
```

Comment:
That was my thinking; now that we have better Clang-side validation and just throw the attr away if it's invalid, this wasn't really nonredundantly testing anything anymore.